pages/topical.vue

```diff
@@ -1,34 +1,47 @@
 <template>
-  <div class="h-full w-full flex flex-wrap gap-4 m-8">
+  <div class="w-full flex flex-wrap gap-4 m-8">
     <div
       v-for="dataset in datasets"
       :key="dataset.title"
-      class="border-4 p-4 prose"
+      class="border-4 p-4 prose space-y-0.5"
+      @click="updateModal(dataset)"
     >
-      <h2>{{ dataset.title }}</h2>
-      <ul>
+      <h2>
+        {{ dataset.title }}
+      </h2>
+      <p>
+        {{ dataset.description }}
+      </p>
+      <ul class="space-y-0.5">
         <li>Contact: {{ dataset.contact[0].name }}</li>
-        <li>License: {{ dataset.license }}</li>
-        <li>Format: {{ dataset.format }}</li>
         <li class="break-words">
           Data access: <a :href="dataset.doi" target="blank">{{ dataset.doi }}</a>
         </li>
       </ul>
-      <h3>Description</h3>
-      <nuxt-content :document="dataset" />
     </div>
+    <ModalDialog v-show="showModal" :dataset="datacard" @close-modal="showModal = false" />
```

Comment:
Maybe just call it `Modal` instead of `ModalDialog`?

Reply:
Corrected!